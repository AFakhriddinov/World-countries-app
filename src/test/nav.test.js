import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Nav from '../components/nav';

describe('Navigation', () => {
  it('Should display correctly', () => {
    const page = renderer.create(
      <Router>
        <Nav />
      </Router>
    );
    const componentTree = page.toJSON();
    expect(componentTree).toMatchSnapshot();
  });
});
