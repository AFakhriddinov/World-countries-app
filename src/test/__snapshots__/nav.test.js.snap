// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Should display correctly 1`] = `
<div
  className="nav"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="back"
      src="back.png"
    />
  </a>
  <h1>
    World Countries
  </h1>
  <div>
    <img
      alt="mic"
      src="mic.png"
    />
    <img
      alt="setting"
      src="setting.png"
    />
  </div>
</div>
`;
